package.json

```diff
@@ -85,6 +90,7 @@
     "@actions/github": "^4.0.0",
     "@babel/core": "^7.11.1",
     "@babel/preset-env": "^7.11.0",
+    "@babel/preset-flow": "^7.12.13",
```

Comment:
It seems required to fix errors thrown by the `react-native-gesture-handler` package. I'm not sure if there's a more correct way to avoid them...

![Screen Shot 2021-03-11 at 8 15 46 AM](https://user-images.githubusercontent.com/32969087/110834737-0186dc80-8242-11eb-844a-61ef46fac2da.png)